packages/@react-spectrum/list/src/listview.css

```diff
@@ -50,31 +53,39 @@
     border-bottom: 0;
   }
 
+  .react-spectrum-ListView--emphasized & {
+    &.is-selected {
+      background-color: var(--spectrum-table-row-background-color-selected);
+      border: 1px solid var(--spectrum-global-color-blue-500);
```

Comment:
border between items is doubling up, maybe omit the border bottom on all but the last item?

Reply:
Pushed a fix to this in https://github.com/adobe/react-spectrum/pull/2517/commits/648711a58ad8d944afdeb6c14e0ed1697109f70f. It is not css-only, but I am open to changing it to be. The selector would be messy with all the divs added by virtualizer.

Reply:
selecting causes the checkbox to shift over by 1px, we'll need to adjust for that https://reactspectrum.blob.core.windows.net/reactspectrum/38a2ac86506dfab52ccdaf8703f90b72e03f754f/storybook-17/index.html?path=/story/listview--selection-single-checkbox-isemphasized
I think it'd be best to implement the border as an inset box shadow, this way when you focus the row, you don't get a 3px border and you also avoid the content shifting

Reply:
Pushed a fix via https://github.com/adobe/react-spectrum/pull/2517/commits/a85a767486a6971e2ddcf7eeaf8be8a0116c1b44, let me know what you think.